CHANGELOG.md

```diff
@@ -1,21 +1,43 @@
+## 2.9.0-beta.1
+
+### Features ✨
+
+* Add `globe` projection. This new projection displays the map as a 3d globe and can be enabled by either passing `projection: globe` to the map constructor or by calling `map.setProjection('globe')`. All layers are supported by globe except for Custom Layers and Sky.
+* Extend atmospheric `fog` with three new style specification properties: `high-color`, `space-color` and `star-intensity` to allow the design of atmosphere around the globe. ([#11590](https://github.com/mapbox/mapbox-gl-js/pull/11590))
+* Add a new line layer paint property in the style specification: `{"line-trim-offset", [trim-start, trim-end]}`, the property will only be effective when `line-gradient` property is set. The line part between `[trim-start, trim-end]` will be marked as transparent to make a line gradient a vanishing effect. The line trim-off percentage is based on the whole line range `[0.0, 1.0]`. If either `trim-start` or `trim-end` offset is out of valid range, the default range `[0.0, 0.0]` will be set. ([#11570](https://github.com/mapbox/mapbox-gl-js/pull/11570))
+* Add an option for providing a geolocation adapter to `GeolocateControl` ([#10400](https://github.com/mapbox/mapbox-gl-js/pull/10400)) (h/t [behnammodi](https://github.com/behnammodi))
+* Add `Map.Title` property to locale options to localise the map `aria-label`. ([#11549](https://github.com/mapbox/mapbox-gl-js/pull/11549)) (h/t [andrewharvey](https://github.com/andrewharvey))
+* Allow duplicated coordinates in tile request URLs. ([#11441](https://github.com/mapbox/mapbox-gl-js/pull/11441)) (h/t [ozero](https://github.com/ozero))
+* Add `source` field requirement to terrain exaggeration in the style specification.  ([#11664](https://github.com/mapbox/mapbox-gl-js/pull/11664))
```

Comment:
I don't think this is a feature. Maybe this should be in Bug Fixes of the style spec changelog?
```suggestion
```